LambdaSchool/CS-Build-Week-1-Flask
# Generates room coordinates within map bounds

# Changelog
### Map

- creates Map class
- adds methods to generate coordinates within map grid
- adds method to create new instance of `Room` at each generated coordinate

### Room
- adds `__repr__` method

### World
- instantiates new `Map`
- assigns rooms generated by `Map` to `World`

### App
- invokes `create_world()` at `/api/check/` endpoint, for testing purposes

#### To test this PR:

1. Start server locally.
2. In `world.py`, uncomment line 63 (`map.print_grid()`) or line 64 (`print(self.rooms)`). 
3. In browser, navigate to `/api/check/`
4. Check your terminal. Depending on what you uncommented, you'll see a grid of blue 0s and yellow 1s, with 1s representing rooms; or you'll see a dictionary of rooms, each of which has `id`, `name`, `description`, and `world_loc` attributes corresponding to the key. 
5. Refresh your browser to see a new grid or new list of rooms.
